__test__/agent.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as http from 'http';
+import * as https from 'http';
+import AgentWrapper from '../src/agent-wrapper';
+
+test('http URL retruns a http Agent', () => {
+	const agentWrapper = new AgentWrapper({});
+
+	const agent = agentWrapper.getAgent('http://no.ne');
+
+	expect(agent).toBeInstanceOf(http.Agent);
+});
+
+test('https URL retruns a https Agent', () => {
```

Comment:
nit these two start with lowercase letter which is slightly inconsistent with rest of the files